rust-runtime/aws-smithy-client/src/test_connection.rs

```diff
@@ -201,6 +201,7 @@ impl<B> TestConnection<B> {
         self.requests.lock().unwrap()
     }
 
+    #[track_caller]
```

Comment:
TIL about [`#[track_caller]`](https://doc.rust-lang.org/reference/attributes/codegen.html#the-track_caller-attribute)